USAGE

```diff
@@ -203,6 +203,11 @@ ABS                                         When this profile is used, TractoFlo
                                             This profile must be used for pathological data.
                                             The aparc+aseg.nii.gz and wmparc.nii.gz must be in the same space than t1.nii.gz
 
+bundling                                    When this profile is used, it will activate custom tracking parameters to improve recobundle results.
+                                            Local tracking will be enable with fa seeding mask.
```

Comment:
with fa seeding and tracking mask

Reply:
Tracking mask is already the case no?
I just want to emphasize on the parameters we change with this profil.